transports/tcp/src/lib.rs

```diff
@@ -889,17 +874,6 @@ mod tests {
         }
 
         fn test(addr: Multiaddr) {
-            #[cfg(feature = "async-io")]
-            {
-                let (ready_tx, ready_rx) = mpsc::channel(1);
-                let listener = listener::<async_io::Tcp>(addr.clone(), ready_tx);
-                let dialer = dialer::<async_io::Tcp>(ready_rx);
-                let listener = async_std::task::spawn(listener);
-                async_std::task::block_on(dialer);
-                async_std::task::block_on(listener);
-            }
-
-            #[cfg(feature = "tokio")]
             {
```

Comment:
scope brackets aren't needed anymore :slightly_smiling_face: (here and in all other places where there was a `{ .. }` scope created for the now removed feature-gating.